samples/zettelkasten.md

```diff
@@ -0,0 +1,3 @@
+source:
+category:
+tag:
```

Comment:
where does normal text content start here? If required/possible add a 

```
------------
```
or empty sep. line here, so it's clear for everybody.